test/dlc_tests/conftest.py

```diff
@@ -417,14 +421,32 @@ def pytest_configure(config):
 
 
 def pytest_runtest_setup(item):
+    """
+    Handle custom markers and options
+    """
+    # Handle quick check tests
+    quick_checks_opts = [mark for mark in item.iter_markers(name="quick_checks")]
+    # Skip quick checks on PR sanity tests, as they will run in separate build-independent job
```

Comment:
can you elaborate on this?

Reply:
We will add a separate CB job to run quick checks, to be executed at the same time as builds instead of after